data-prepper-plugins/peer-forwarder/build.gradle

```diff
@@ -15,9 +15,16 @@ dependencies {
     implementation "io.opentelemetry:opentelemetry-proto:${versionMap.opentelemetry_proto}"
     implementation "com.linecorp.armeria:armeria:1.9.1"
     implementation "com.linecorp.armeria:armeria-grpc:1.9.1"
+    implementation "com.amazonaws:aws-java-sdk-s3:1.12.15"
+    implementation "com.amazonaws:aws-java-sdk-acm:1.12.12"
+    implementation "commons-io:commons-io:2.10.0"
+    implementation "org.apache.commons:commons-lang3:3.12.0"
     implementation "commons-validator:commons-validator:1.7"
```

Comment:
Not sure if we should prioritize this now but I think AWS SDK v2 will be the way forward. Maybe we switch to in the future release.

Reply:
Moving to AWS SDK v2 is a bigger effort. We can take care as part of separate PR.